extensions/notebook/src/extension.ts

```diff
@@ -113,6 +125,10 @@ export async function activate(extensionContext: vscode.ExtensionContext): Promi
 	extensionContext.subscriptions.push(vscode.window.registerTreeDataProvider(BOOKS_VIEWID, bookTreeViewProvider));
 	extensionContext.subscriptions.push(vscode.window.registerTreeDataProvider(READONLY_BOOKS_VIEWID, untitledBookTreeViewProvider));
 
+	extensionContext.subscriptions.push(vscode.commands.registerCommand('notebook.command.openCreatedBook', async (bookPath: string) => {
```

Comment:
I don't see the value in making a new command - just run the bookTreeView.openBook command and pass it `bookPath, false, undefined`